core/whistle/src/wh_util.erl

```diff
@@ -632,9 +633,9 @@ put_callid(JObj) -> erlang:put('callid', callid(JObj)).
 get_callid() -> erlang:get('callid').
 
 callid(Prop) when is_list(Prop) ->
-    props:get_first_defined([<<"Call-ID">>, <<"Msg-ID">>], Prop, ?LOG_SYSTEM_ID);
+    props:get_first_defined([?KEY_LOG_ID, <<"Call-ID">>, ?KEY_MSG_ID], Prop, ?LOG_SYSTEM_ID);
```

Comment:
@jamesaimonetti that defeats the purpose as you will always get Msg-ID in `jonny5` sync. `System-Log-ID` shyould come first because it is only used in `wh_amqp_worker` and fetched from the payload by  `gen_listener`